docs/src/dedicated-gen-3/security.md

```diff
@@ -1,59 +1,95 @@
+1. Isolating the affected systems.
+2. Collecting forensic evidence for later analysis, including a byte-for-byte copy of the affected systems.
+3. Restoring normal operations.
 
-Platform.sh cooperates with relevant law enforcement, and inform law enforcement in the event of an attempted malicious intrusion.
-Depending on the type of incident the root cause analysis may be conducted by law enforcement rather than Platform.sh personnel.
+Once normal service is restored, a root cause analysis is performed to determine exactly what happened.
+Upon request, Platform.sh can provide you with a Reason for Outage report that summarizes the incident, cause, and steps taken.
 
-Platform.sh endeavors to notify affected customers within 24 hours in case of a personal data breach and 72 hours in case of a project data breach.
+Platform.sh cooperates with relevant law enforcement,
+and informs law enforcement in the event of an attempted malicious intrusion.
+Depending on the type of incident, the root cause analysis may be conducted by law enforcement rather than Platform.sh personnel.
+
+Platform.sh endeavors to notify affected customers within 24 hours in case of a personal data breach
+and 72 hours in case of a project data breach.
 
 <!-- vale Vale.Spelling = NO -->
 <!-- Spelling off because of the French-->
-Under the European General Data Protection Regulation (GPDR), Platform.sh is required to notify our supervising authority within 72 hours of a discovered breach that may result in risk to the rights and freedoms of individuals.
-Our supervising authority is the French [Commission Nationale de l'Informatique et des Libertés](https://www.cnil.fr/).
+Under the European General Data Protection Regulation (GPDR),
+Platform.sh is required to notify its supervising authority within 72 hours of a discovered breach
+that may result in risk to the rights and freedoms of individuals.
+The supervising authority for Platform.sh is the French [Commission Nationale de l'Informatique et des Libertés](https://www.cnil.fr/).
 <!-- vale Vale.Spelling = YES -->
 
-## Audit trail
+### Audit trail
 
-As part of the security incident process we record a log of all steps taken to identify, isolate, and respond to the incident.
+As part of the security incident process, Platform.sh records a log of all steps taken to identify,
+isolate, and respond to the incident.
 This log may include:
 
-* A byte-for-byte copy of the affected systems
-* How the intrusion was detected
-* The steps taken to contain the intrusion
-* Any contact with 3rd parties, including law enforcement
-* Any conclusions reached regarding the root cause
+- A byte-for-byte copy of the affected systems
+- How the intrusion was detected
+- The steps taken to contain the intrusion
+- Any contact with third parties, including law enforcement
+- Any conclusions reached regarding the root cause
 
 ## Encryption
```

Comment:
@remilejeune2 Not sure we want to keep this section? Or maybe we should change it to provide more generic info instead of a per cloud provider list (with only 2 providers for now...)?